pipeline/frontend/yaml/container_test.go

```diff
@@ -87,7 +87,6 @@ func TestUnmarshalContainer(t *testing.T) {
 		ExtraHosts:    []string{"somehost:162.242.195.82", "otherhost:50.31.209.229"},
 		Image:         "golang:latest",
 		Isolation:     "hyperv",
-		Labels:        types.SliceorMap{"com.example.type": "build", "com.example.team": "frontend"},
```

Comment:
Unrelated removal. This should be part of `step.backend_options.docker`

Reply:
well I can add it back ... it was in my way and was never documented so it got removed

I'll create a pull up-ahead to remove that

Reply:
#1825